### ISSUE TITLE
[a11y] Inconsistent focus outlines in status bar

### ISSUE THREAD

**Issue by IsabelParedes** [jupyterlab/jupyterlab#18584]
## Description

Currently, the elements in the status bar have different styles for the focus outline which are inconsistent with the other elements in the application. This makes it difficult to see which element is focused when navigating with a keyboard.

The `Simple` switch uses
https://github.com/jupyterlab/jupyterlab/blob/63dabd5562f5b2537db60ababc667f13494da8d8/packages/application/style/buttons.css#L24-L27

The `1px` width is barely visible.

<img width="2026" height="72" alt="Image" src="https://github.com/user-attachments/assets/bed72c88-da14-4eb6-a360-98d2a9f4a1cb" />

<br>

The `GroupItem`s use the browser's default settings which can result in black/white outlines.
```css
:focus-visible
{
    outline: -webkit-focus-ring-color auto 1px;
}
```

<img width="2026" height="72" alt="Image" src="https://github.com/user-attachments/assets/09ae4fd1-1dc4-45b0-b433-712684578a5a" />

<img width="2026" height="72" alt="Image" src="https://github.com/user-attachments/assets/4a7a15c9-271b-4e3a-8925-70f5664bb86b" />

## Reproduce

1. Launch JupyterLab
2. Press `Tab` multiple times to navigate to the status bar

## Expected behavior

The items in the status bar should have focus outlines such as the ones used in the `SideBar` which are clearly visible.

https://github.com/jupyterlab/jupyterlab/blob/63dabd5562f5b2537db60ababc667f13494da8d8/packages/application/style/sidepanel.css#L55-L59

<img width="79" height="199" alt="Image" src="https://github.com/user-attachments/assets/bf16926a-f797-4af5-9491-5e00ec1c46e3" />

## Context

- Operating System and version: Garuda Linux
- Browser and version: Chromium v145.0.7632.116
- JupyterLab version: 4.6.0a3

**Comment by krassowski** [comment]
Closing as fixed by https://github.com/jupyterlab/jupyterlab/pull/18585
